src/Init/Data/BitVec/Basic.lean

```diff
@@ -669,6 +669,21 @@ def ofBoolListLE : (bs : List Bool) → BitVec bs.length
 | [] => 0#0
 | b :: bs => concat (ofBoolListLE bs) b
 
+/-! ## Overflow -/
+
+/-- Overflow predicate for unsigned addition modulo 2^w.
```

Comment:
```suggestion
/-- `uaddOverflow x y` returns `true` if addition of `x` and `y` results in *unsigned* overflow.
```

I like to be explicit in docstrings, and "Overflow predicate" feels a little vague still